armi/physics/fuelCycle/fuelHandlers.py

```diff
@@ -212,10 +213,19 @@ def _compareAssem(candidate, current):
     @staticmethod
     def _getParamMax(a, paramName, blockLevelMax=True):
         """Get parameter with Block-level maximum."""
+        volIntegratedParamNames = [
+            p.name
+            for p in a.getBlocks()[0].p.paramDefs.atLocation(
+                ParamLocation.VOLUME_INTEGRATED
+            )
+        ]
```

Comment:
Okay, so I see what you're doing here. You are getting _all_ the volume-integrated parameters and then checking to see if the `"paramName"` passed into this method is in that list.

But why not just check if that one parameter is volume-integrated?

```python
a.getBlocks()[0].p.paramDefs[paramName].location == ParamLocation.VOLUME_INTEGRATED
```

Does my way not work? Am I missing something?

Reply:
I was scrounging the logic from elsewhere, probably the logic for growing to full core. I've updated the logic here.